lib/packer.js

```diff
@@ -15,3 +27,28 @@ function encodeDate(obj) {
 function decodeDate(buf) {
   return new Date(buf.toString('utf8'));
 }
+
+function jsonFormat() {
+  var packer = {
+    encode: function(value) {
+      return JSON.stringify (value);
```

Comment:
@bajtos 

I think it can be an encoding option based on [the Buffer encodings](https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings), such as `base64`, `hex`, and etc.